pydelhi.py

```diff
@@ -501,30 +501,19 @@ def help(bot, update):
 
 To contribute to|modify this bot : https://github.com/realslimshanky/PyDelhi-Bot
 ''')
+#pass_args=True
+#dispatcher.add_handler(CommandHandler(    'modifyteam', modifyteam, pass_args=True))
 
+#dispatcher.add_handler(CommandHandler('start', start, pass_args=True))
+commands=[help,modifyteam,teams,start,sponsorshipteam,venueteam,vendorteam,mailinglist,website,cfpteam,irc,twitter,meetuppage,nextmeetup,nextmeetupschedule,facebook,github,invitelink,socialmediateam,logisticsteam,websiteteam,designteam]
+
+for command in commands:
+    
+    if command != "modifyteam" or command != "start":
```

Comment:
You can create two arrays - arg_commands normal_commands. Even better and more extensible create a dict.